### PR TITLE
Fixes #7834: Adapt mulriserver setup scripts to the documentation

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7834

### CODE REVIEW

File: scripts/rudder-multiserver-setup/rudder-relay-top.sh [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
 # This is copied from http://www.rudder-project.org/rudder-doc-2.11/rudder-doc.html#relay-servers
-if [ "$OS" = "RHEL" ] ; then
-  $PM_COMMAND rudder-agent httpd rsyslog
-elif [ "$OS" = "UBUNTU" -o "$OS" = "DEBIAN" ] ; then
-  $PM_COMMAND rudder-agent apache2 apache2-utils rsyslog
+if [ "${OS}" = "RHEL" ] ; then
+  ${PM_COMMAND} rudder-agent httpd rsyslog
+elif [ "${OS}" = "UBUNTU" -o "${OS}" = "DEBIAN" ] ; then
+  ${PM_COMMAND} rudder-agent apache2 apache2-utils rsyslog
   echo "Now fix the bug on /var/lib/dpkg/info/rudder-agent.postinst"
```
Comment (jooooooon) [inline scripts/rudder-multiserver-setup/rudder-relay-top.sh]: Do you know what bug this is? I know it's not new in this PR, but if we could have an explanation in this script that would help *a lot*

File: scripts/rudder-multiserver-setup/rudder-relay-top.sh [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
+if [ "${OS}" = "RHEL" ] ; then
+  ${PM_COMMAND} rudder-agent httpd rsyslog
+elif [ "${OS}" = "UBUNTU" -o "${OS}" = "DEBIAN" ] ; then
+  ${PM_COMMAND} rudder-agent apache2 apache2-utils rsyslog
   echo "Now fix the bug on /var/lib/dpkg/info/rudder-agent.postinst" 
   echo "Then run aptitude install" 
   bash -i
   a2enmod dav dav_fs
-  if [ "$A2_VERSION" = 2.4 ] ; then
+  if [ "${A}2_VERSION" = 2.4 ] ; then
```
Comment (jooooooon) [inline scripts/rudder-multiserver-setup/rudder-relay-top.sh]: I think you quoted this wrong. But even so, where does this variable come from?

Comment (jooooooon) [inline scripts/rudder-multiserver-setup/rudder-relay-top.sh]: Er, not quoted, but used braces wrong

File: scripts/rudder-multiserver-setup/rudder-web.sh [hunk 3]
```diff
@@ -40,7 +50,7 @@ setenforce 0
 
 # Now, accept the other three servers in the web interface (it may take up to 5 minutes for them to appear in "Accept new nodes")
 # Then, use the UUID of server 1 (rudder-front) we stored above and run this command (from /opt/rudder/etc/uuid.hive)
-/opt/rudder/bin/rudder-node-to-relay ${FRONT_UUID}
+#/opt/rudder/bin/rudder-node-to-relay ${FRONT_UUID}
```
Comment (jooooooon) [inline scripts/rudder-multiserver-setup/rudder-web.sh]: Why are you commenting this?

**Comment by jooooooon** [comment]
Ping @peckpeck

File: scripts/rudder-multiserver-setup/rudder-relay-top.sh (hunk 1 above)
Comment (peckpeck) [inline scripts/rudder-multiserver-setup/rudder-relay-top.sh]: I don't remember what bug this is refering to, but I think it must have been fixed a long time ago.

File: scripts/rudder-multiserver-setup/rudder-relay-top.sh (hunk 2 above)
Comment (peckpeck) [inline scripts/rudder-multiserver-setup/rudder-relay-top.sh]: Brace wrong, 2.4 should be quoted and A2_version comes from a few lines below :)

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
Fixed except for the bug comment.

File: scripts/rudder-multiserver-setup/rudder-relay-top.sh (hunk 1 above)
Comment (jooooooon) [inline scripts/rudder-multiserver-setup/rudder-relay-top.sh]: I would hope it has been fixed, yes.

We should test again, and fix it if not.

For the mean time, let's remove this.

**Comment by jooooooon** [comment]
Aside from that, ready for merge

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**